doc/mob_skill_db.conf

```diff
@@ -0,0 +1,168 @@
+	"MSS_IDLE" //In Idle State
+	"MSS_WALK" //While Movement
+	"MSS_DEAD" //While Dying
+	"MSS_FOLLOW //Mob Following a player without being attacked
+
+SkillTarget: The Target who the mob will target with the skill
+	"MST_SLAVELE" //Slave
+	"MST_SELF" //On Himself
+	"MST_TARGET" //On Target
+	"MST_RANDOM" //RANDOM TARGET!
+	"MST_FRIEND" //On friend (Near Monster or so on like the Geographer Heal)
+	"MST_MASTER" //Only for Slaves which target the Master for Example with AL_HEAL to heal him.
+	"MST_AROUND5"
+	"MST_AROUND6"
+	"MST_AROUND7"
+	"MST_AROUND8"
+	"MST_AROUND1"
+	"MST_AROUND2"
+	"MST_AROUND3"
+	"MST_AROUND4"
+	"MST_AROUND" //= MST_AROUND4,
+
+
+Rate: Monster Rate for Casting the Skill, 1 = 0.01%. 10= 0,1%, 100% = 1% ...
+
+CastTime: time in ms how long the mob need to cast the skill 1000ms = 1sec
+
+Delay: How much time will be between the use and the next use for the skill in ms 1000ms = 1 sec.
+
+Cancelable: if it's possible for the player to cancel the skill while the mob is casting
+	    If set to "True" the skill can be interrupted at casting state
+
+CastCondition: Which Condition is needed to cast the skill
+	"MSC_MYHPLTMAXRATE" //When number of HP in Rate is less than or equal to the original specified number.
+	"MSC_MYHPINRATE" //HP Between two rates where the skills is casted, ConditionData has be the lower value and val0 the higher value
+	"MSC_SPAWN" //When mob spawns/respawns (no condition value).
+	"MSC_ALWAYS" //Always
+	"MSC_RUDEATTACKED" //When mob is rude attacked (no condition value).
+	"MSC_SKILLUSED" //if an player uses an skill, ConditionData can set for a specific skill like MG_SIGHT
+	"MSC_AFTERSKILL" //Nach benutzung eines Skills
+	"MSC_ATTACKPCGE" //When attack PCs become greater than or equal to the specified number.
+	"MSC_ATTACKPCGT
+	"MSC_FRIENDHPLTMAXRATE" //When mob's friend's HP drops to the specified %.
+	"MSC_FRIENDHPINRATE"//When mob's friend's HP drops between two HP Rates in %
+	"MSC_MYSTATUSON" //If mob has the specified abnormality in status.(Example: ConditionData "SC_HIDING")
+	"MSC_MYSTATUSOFF" //If mob specified abnormality in status has ended .(Example: ConditionData "SC_HIDING")
+	"MSC_SLAVELE" //When number of slaves is less than or equal to the original specified number.
+	"MSC_SLAVELT" //When number of slaves is between two numbers.
+	"MSC_LONGRANGEATTACKED" //When long range attacked, ex. bows, guns, ranged skills (no condition value).
+	"MSC_CLOSEDATTACKED" //When Melee Attacked
+	"MSC_FRIENDSTATUSON" //If friend has the specified abnormality in status.
+	"MSC_FRIENDSTATUSOFF" //If friends abnormality in status has ended
+	"MSC_CASTTARGETED //When a target is in cast range (no condition value).
+	"MSC_MASTERHPLTMAXRATE" //When number of Master's HP in Rate is less than or equal to the original specified number.
+	"MSC_MASTERATTACKED" //if the master is attacked
+	"MSC_ALCHEMIST" //If summoned by Alchemist like Marine Sphere
+
+ConditionData:
```

Comment:
The ConditionData as well as the val0~val4 below could be worded more simply as `Additional parameters used by the CastCondition, where applicable`, and then in the above documentation about each CastCondition it should say whether it uses additional parameters and their meaning

Reply:
Okay will adding them soon.